pennylane/devices/default_qubit.py

```diff
@@ -626,12 +626,9 @@ def _apply_state_vector(self, state, device_wires):
             raise ValueError("State vector must be of length 2**wires.")
 
         norm_error_message = "Sum of amplitudes-squared does not equal one."
-        if qml.math.get_interface(state) == "torch":
+        if not hasattr(qml.math.linalg.norm(state, ord=2), "full_lower"):
```

Comment:
What does `full_lower` mean :)

Reply:
@rmoyard I have a worry that this might not be explicit enough. Maybe something like

```python
if qml.math.get_interface(state) == "jax" and not hasattr(...):
```

might be safer, in that the if statement will only explicitly apply in the JAX case?

Reply:
I agree that it is not clear at all, I will separate the cases!

Reply:
> From Jax documentation, full_lower is an optional optimization so that we unbox values out of Tracers as much as possible.

Nice, I wasn't aware of this function!